# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/qemu/hotfix-sparc64.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- qemu-9.2.3/tcg/tcg.c.vanilla	2025-04-13 13:36:34.836082637 +0200
+++ qemu-9.2.3/tcg/tcg.c	2025-04-13 13:41:44.088271256 +0200
@@ -4165,7 +4165,7 @@
         tcg_raise_tb_overflow(s);
     }
     s->current_frame_offset = off + size;
-#if defined(__sparc__)
+#if defined(TCG_TARGET_STACK_BIAS)
     off += TCG_TARGET_STACK_BIAS;
 #endif
 
